.github/workflows/test.yml

```
@@ -45,25 +45,19 @@ jobs:
-        version: latest
-        virtualenvs-create: true
-        virtualenvs-in-project: true
-    
+
+    - name: Install UV
+      run: pip install uv
+
     - name: Load cached venv
-      id: cached-poetry-dependencies
+      id: cached-uv-dependencies
       uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)